src/FNAPlatform/FNAPlatform.cs

```diff
@@ -93,6 +93,7 @@ static FNAPlatform()
 
 			Malloc =			SDL2_FNAPlatform.Malloc;
 			Free =				SDL2.SDL.SDL_free;
+			SetEnv =			SDL2_FNAPlatform.SetEnv;
```

Comment:
Looks like we'll have to set this above the args checks, otherwise we'll get null references.